docs/index.rst

```diff
@@ -1,106 +1,79 @@
-.. Toil documentation master file, created by
-   sphinx-quickstart on Tue Aug 25 12:37:16 2015.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 Toil
 ====
 
-Toil is a workflow engine entirely written in Python. It features:
-
-* Easy installation, e.g. ``pip install toil``.
-
-* `Common Workflow Language`_ (`CWL`_) support
-
-  Nearly complete support for the stable CWL v1.0 specification,
-  allowing it to execute CWL workflows.
-
-* Cross platform support
-
-  Develop and test on your laptop then deploy on any of the following:
-
-  - Commercial clouds:
-    - `Amazon Web Services`_ (including the `spot market`_)
-    - `Microsoft Azure`_
-    - `Google Compute Engine`_
-  - Private clouds:
-    - `OpenStack`_
-  - High Performance Computing Environments:
-    - `GridEngine`_
-    - `Apache Mesos`_
-    - `Parasol`_
-    - Individual multi-core machines
-
-* A small API
-
-  Easily mastered, the Python user API for defining and running workflows is built upon one core class.
-
-* Complete file and stream management:
-
-  Temporary and persistent file management that abstracts the details of the
-  underlying file system, providing a uniform interface regardless of
-  environment. Supports both atomic file transfer and streaming interfaces, and
-  provides encryption of user data.
-
-* Scalability:
-
-  Toil can easily handle workflows concurrently using hundreds of nodes and
-  thousands of cores.
-
-* Robustness:
-
-  Toil workflows support arbitrary worker and leader failure, with strong
-  check-pointing that always allows resumption.
-
-* Efficiency:
+Toil is an open-source pure-Python workflow engine that lets people write better
+pipelines. You can:
 
-  Caching, fine grained, per task, resource requirement specifications, and
-  support for the AWS spot market mean workflows can be executed with little
-  waste.
+* Write your workflows in `Common Workflow Language`_ (CWL),
+* Run workflows on your laptop or on huge commercial clouds such as
+  `Amazon Web Services`_ (including the `spot market`_), `Microsoft Azure`_,
+  `OpenStack`_, and `Google Compute Engine`_,
+* Take advantage of high-performance computing environments with batch systems
+  like `GridEngine`_, `Apache Mesos`_, and `Parasol`_,
+* Run workflows concurrently at scale using hundreds of nodes and thousands of cores,
```

Comment:
Can you add a link to our paper pre-print here? http://biorxiv.org/content/early/2016/07/07/062497